### ISSUE TITLE
Support route for one movie

### ISSUE THREAD

**Issue by SirSerje** [SirSerje/eight-millimeter-shelter#8]
As user I want to open localhost:3678/movie/256 and see page with one movie.

**Issue blocked by #7** 

- [ ] Backend should be updated
- [ ] Frontend should have new page design
- [ ] Postman schema should be updated

**Comment by SirSerje** [comment]
in future, nice to have - support more deep routes for movies, like host/some/path/?id=21
